openpype/settings/defaults/project_settings/nuke.json

```diff
@@ -19,16 +19,21 @@
             "rules": {}
         },
         "viewer": {
-            "viewerProcess": "sRGB (default)"
+            "viewerProcess": "sRGB (ACES)"
         },
         "baking": {
             "viewerProcess": "rec709 (default)"
```

Comment:
```suggestion
            "viewerProcess": "Rec.709 (ACES)"
```

Reply:
But it should rather be `ACES/Rec.709` right? Just to make it aligned with Hiero display/view name convention.